ext/semian/resource.h

```diff
@@ -21,7 +21,7 @@ int system_max_semaphore_count;
  * Creates a new Resource. Do not create resources directly. Use Semian.register.
  */
 VALUE
-semian_resource_initialize(VALUE self, VALUE id, VALUE tickets, VALUE permissions, VALUE default_timeout);
+semian_resource_initialize(VALUE self, VALUE id, VALUE tickets, VALUE quota, VALUE permissions, VALUE default_timeout);
```

Comment:
Yeah i think I had a fixme for that, at some point.

Regarding the name, initialize_semaphore I'm on the fence as this handles more than *just* semaphore initialization. The rest of the functions use semian_resource as a prefix, and they are all associated with the rubyland semian resource object. Then again, as far as rubyland is concerned this is all just the magic semaphore code... so I don't really care what we call it.